k_means_constrained/k_means_constrained_.py

```diff
@@ -32,7 +32,8 @@
 def k_means_constrained(X, n_clusters, size_min=None, size_max=None, init='k-means++',
                         n_init=10, max_iter=300, verbose=False,
                         tol=1e-4, random_state=None, copy_x=True, n_jobs=1,
-                        return_n_iter=False):
+                        return_n_iter=False, distance_metric = euclidean_distances, 
+                        **kwargs):
```

Comment:
Could you change **kwargs to a argument that accepts an optional dictionary of kwargs for the distance metric 
```suggestion
                        distance_metric_kwargs = None):
```